libr/anal/p/anal_x86_cs.c

```diff
@@ -641,23 +650,23 @@ static void anop_esil (RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 		// TODO: RCL Still does not work as intended
 		//  - Set flags
 		{
-			char *src = getarg (&gop, 1, 0, NULL);
-			char *dst = getarg (&gop, 0, 0, NULL);
+			char src[64];
```

Comment:
define src and dst once at the begining of the function instead of doing it in aevery stackframe